src/components/Tabs.tsx

```diff
@@ -0,0 +1,37 @@
+import cn from 'classnames';
+import { Link } from 'react-router-dom';
+import { Tab } from '../types/Tab';
+
+type Props = {
+  tabs: Tab[];
+  selectedTab?: string;
+};
+
+export const Tabs: React.FC<Props> = ({ tabs, selectedTab }) => {
+  const isSelected = (tab: Tab) => tab.id === selectedTab;
+
+  return (
+    <>
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map((tab) => (
+            <li
+              key={tab.id}
+              data-cy="Tab"
+              className={cn({ 'is-active': isSelected(tab) })}
```

Comment:
You don't really need to create a function for it 
```suggestion
              className={cn({ 'is-active': tab.id === selectedTab })}
```

Reply:
Fixed